mmf/common/test_reporter.py

```diff
@@ -123,12 +130,12 @@ def __len__(self):
     def __getitem__(self, idx):
         return self.current_dataset[idx]
 
-    def add_to_report(self, report, model):
+    def add_to_report(self, report, model, master_only=True):
```

Comment:
Users might find this name a bit confusing and how it differentiates with `is_master`. Can we make it more succinct  with something like `execute_on_master_only`? Thoughts?

Reply:
(changed to execute_on_master_only)